src/app/shared/helper-services/export-to-template.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http'
+import * as XLSX from 'xlsx';
+import { UtilityMeterdbService } from 'src/app/indexedDB/utilityMeter-db.service';
+import { IdbUtilityMeter } from 'src/app/models/idb';
+@Injectable({
+  providedIn: 'root'
+})
+export class ExportToTemplateService {
+
+  constructor(private http: HttpClient, private utilityMeterDbService: UtilityMeterdbService) { }
+
+  exportToTemplate() {
+    this.http.get("assets/csv_templates/VERIFI-Import-Data.xlsx", { responseType: "arraybuffer" }).subscribe(
+      data => {
+        console.log(data);
+        let blob = new Blob([data], { type: 'application/mx-excel' });
+        this.readWorkbook(blob);
+      },
+      error => {
+        console.log("ERROROROOROR")
```

Comment:
Still need console logging? multiple in this service

Reply:
This service was unused. deleted